tests/unit/hive_metastore/test_table_migrate.py

```diff
@@ -1375,63 +1375,70 @@ def test_revert_migrated_tables_failed(caplog, mock_pyspark):
     assert "Failed to revert table hive_metastore.test_schema1.test_table1: error" in caplog.text
 
 
-def test_refresh_migration_status_published_remained_tables(caplog, mock_pyspark):
-    backend = MockBackend()
+def test_refresh_migration_status_check_remaining_tables(ws, mock_backend, caplog) -> None:
     table_crawler = create_autospec(TablesCrawler)
-    client = mock_workspace_client()
-    table_crawler.snapshot.return_value = [
-        Table(
-            object_type="EXTERNAL",
-            table_format="DELTA",
-            catalog="hive_metastore",
-            database="schema1",
-            name="table1",
-            location="s3://some_location/table1",
-            upgraded_to="ucx_default.db1_dst.dst_table1",
+    tables = (
+        (
+            Table(
```

Comment:
The was part of factoring out the sequence of tables returned by the `snapshot()` mock (and make it immutable) so that the migration-index mock return the correct values. I'm not sure why the extra parentheses were in there, but I've removed them. Other than those it was only whitespace.